README.md

```diff
@@ -2,9 +2,8 @@ Puppet Strings
 ==============
 [![Build Status](https://travis-ci.org/puppetlabs/puppet-strings.png?branch=master)](https://travis-ci.org/puppetlabs/puppet-strings) [![Gem Version](https://badge.fury.io/rb/puppet-strings.svg)](https://badge.fury.io/rb/puppet-strings)
 
-A Puppet command built on [YARD](http://yardoc.org/).
 
-Puppet Strings generates HTML documentation for Puppet extensions written in Puppet and Ruby.
+Puppet Strings creates documentation for Puppet code by reading code and [YARD](http://yardoc.org/)-style code comments and generating HTML, JSON, or Markdown output.
```

Comment:
Hmmm, it creates docs for Ruby code, also. 
What about something like:

> Puppet Strings is a Puppet face that extends [YARD](http://yardoc.org) to read Puppet code and YARD-style comments, then generate HTML, JSON, or Markdown documentation.

Mentioning it as a YARD extension implies Ruby functionality as well.

Reply:
I tried something else, because I didn't want to talk about faces, and I want to talk about what it *does* rather than what it *is*. See what you think.